lib/fastlane/plugin/test_center/actions/tests_from_xctestrun.rb

```diff
@@ -16,14 +16,19 @@ def self.xctestrun_tests(xctestrun_path, invocation_based_tests)
           next if ignoredTestables.include? testable_name
 
           xctest_path = xctest_bundle_path(xctestrun_rootpath, xctestrun_config)
-          test_identifiers = XCTestList.tests(xctest_path)
-          UI.verbose("Found the following tests: #{test_identifiers.join("\n\t")}")
+          if xctestrun_config.key?('OnlyTestIdentifiers')
+            test_identifiers = xctestrun_config['OnlyTestIdentifiers']
+            UI.verbose("Identifiers after adding onlytest tests: #{test_identifiers.join("\n\t")}")
+          end
 
           if xctestrun_config.key?('SkipTestIdentifiers')
+            test_identifiers = XCTestList.tests(xctest_path)
```

Comment:
We still need the tests if there are no `SkipTestIdentifiers`

Reply:
Then it does not make sense that i move OnlyTestIdentifiers block above this line: test_identifiers = XCTestList.tests(xctest_path) (as per your comment in issue), because this line will overwrite test_identifiers anyway.